cilantro_audit/widgets/view_audit_templates.kv

```diff
@@ -0,0 +1,110 @@
+#:kivy 1.11.1
+
+
+Manager:
+
+
+<ThisPage>:
+    name: 'this_page'
+
+    BoxLayout:
+        orientation: 'vertical'
+
+        BoxLayout:
+            name: 'header'
+            size_hint: 1, 0.1
+
+            orientation: 'horizontal'
+
+            canvas.before:
+                Color:
+                    rgb: 0.275, 0.314, 0.275, 1
+                Rectangle:
+                    size: self.size
+                    pos: self.pos
+
+            Button:
+                text: 'Back'
+                size_hint: 0.3, 1.0
+                on_release:
+                    app.root.current = 'AuditorPage'
+                    app.root.transition.duration = 0.3
+                    app.root.transition.direction = 'right'
+
+            Label:
+                text: 'Audit Templates'
+                color: 1.0, 1.0, 1.0, 1
```

Comment:
Could be larger font for visibility.

Reply:
Should be the same size as button text. At least for my machine that's what I see. Could you double check if you still see it small, or did you mean you want it even larger than the rest of text fields on screen?

![screenshot 0](https://user-images.githubusercontent.com/18043111/67999899-44cb4700-fc1b-11e9-93ab-b57b57f57499.png)